vpplink/generated/vpp_clone_current.sh

```diff
@@ -96,6 +96,7 @@ function git_clone_cd_and_reset ()
 git_clone_cd_and_reset "$1" cfa0953251cbab435307baf3dcd249fd95afaf1f # misc: VPP 24.10 Release Notes
 
 git_cherry_pick refs/changes/26/34726/3 # 34726: interface: add buffer stats api | https://gerrit.fd.io/r/c/vpp/+/34726
+git_cherry_pick refs/changes/94/41094/20 # 41094: vlib: improve core pinning | https://gerrit.fd.io/r/c/vpp/+/41094
```

Comment:
maybe 24th patchset, and btw this got upstreamed

Reply:
Good point, updated